packages/ra-core/src/form/sanitizeEmptyValues.spec.ts

```diff
@@ -1,77 +0,0 @@
-import expect from 'expect';
-import sanitizeEmptyValues from './sanitizeEmptyValues';
-
-describe('sanitizeEmptyValues', () => {
```

Comment:
We should have unit tests on the Form component to make sure the behavior stays the same without this sanitization